src/emulator/eventarcEmulator.ts

```diff
@@ -95,17 +112,33 @@ export class EventarcEmulator implements EmulatorInstance {
       });
     };
 
+    const publishNativeEventsRoute = `/google/publishEvents`;
```

Comment:
Same here - the native events handler is essentially doing the same job as the custom events handler.